### ISSUE TITLE
in my firefox profile i found that i have db9 and there is no db8

### ISSUE THREAD

**Issue by mrhili** [pyllyukko/user.js#493]

**Comment by pyllyukko** [comment]
Indeed. See [this answer](https://github.com/pyllyukko/user.js/issues/495#issuecomment-684948670).

**Comment by nodiscc** [comment]
`cas.sh` will be dropped in https://github.com/pyllyukko/user.js/pull/514

**Comment by nodiscc** [comment]
https://github.com/pyllyukko/user.js/pull/514 merged, this issue should be closed
